packages/frontend/core/src/components/affine/auth/oauth.tsx

```diff
@@ -50,8 +50,9 @@ function OAuthProvider({ provider }: { provider: OAuthProviderType }) {
 
   const onClick = useCallback(() => {
     let oauthUrl =
-      (BUILD_CONFIG.isElectron ? BUILD_CONFIG.serverUrlPrefix : '') +
-      `/oauth/login?provider=${provider}`;
+      (BUILD_CONFIG.isElectron || BUILD_CONFIG.isIOS || BUILD_CONFIG.isAndroid
```

Comment:
isApp?

Reply:
I want to keep one flag per entry for now, lets see if we need to add an isApp in the future

Reply:
just make `isApp` a shortcut for `BUILD_CONFIG.isElectron || BUILD_CONFIG.isIOS || BUILD_CONFIG.isAndroid` otherwise it's too verbose